elastic/kibana
# [Synthetics] allow disabling throttling for project monitors

## Summary

Resolves https://github.com/elastic/kibana/issues/148344

Allows for disabling throttling by adjusting the request types for project monitors

### Testing
1. Checkout the synthetics agent PR. Be sure to link the development version of the package to the your test project directory via `npm link`. If you don't already have a test directory, create one via `npm run init`.
2. In a test project monitor, set `monitor.use({ throttling: false })`. 
3. Ensure no errors are reported back to the synthetics agent. 
4. In Uptime, navigate to the edit monitor page for the monitor you're testing. Scroll down to Advanced browser options. Open the accordion and confirm throttling is disabled 
<img width="1752" alt="Screen Shot 2023-01-10 at 2 20 24 PM" src="https://user-images.githubusercontent.com/11356435/211642237-de2d0ba3-2822-4ad4-beb4-a4404b9caae9.png">
